src/sync/installation.ts

```diff
@@ -275,9 +287,18 @@ async function doProcessInstallation(app, data: BackfillMessagePayload, sentry:
 		const { edges, jiraPayload } = await execute();
 		if (jiraPayload) {
 			try {
-				await jiraClient.devinfo.repository.update(jiraPayload, {
-					preventTransitions: true
-				});
+				switch (task) {
+					case "build":
+						await jiraClient.workflow.submit(jiraPayload);
+						break;
+					case "deployment":
+						await jiraClient.deployment.submit(jiraPayload);
+						break;
+					default:
+						await jiraClient.devinfo.repository.update(jiraPayload, {
+							preventTransitions: true
+						});
```

Comment:
I'm not sure I understand this point of the switch statement.

Reply:
For all current tasks except build/deployment we use the `/rest/devinfo/0.10/bulk` endpoint, however deployment and build need to hit a different path so we are just handling where to send it based on task... could convert it to a function to return the appropriate function instead....?

Reply:
oh shoot, I didn't see it was hitting workflow vs deployment in my late night haze, nvm, carry on :)